src/main/java/coydir/model/person/Person.java

```diff
@@ -25,12 +25,15 @@ public class Person {
     private final Position position;
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
+    private final Set<Leave> leaves = new HashSet<>();
```

Comment:
Is there a reason why `Set<>` is used? If we are keeping track of leaves with some concept of ordering/sequence, would `ArrayList<>` not be better?

Reply:
Hmm. You raise a good point. What do yall think? @ngshijun @Kok-je @chao890

Reply:
I think it does not matter that much as the leaves probably will not always be added in chronological order, furthermore set could prevent duplicate leaves being recorded, so I think using `HashSet<>` might be better.

Reply:
Eh actually looking at this I just had another thought. Should they not be ordered chronologically? So perhaps a priority queue? 

In this case, if I add two different leave periods for an employee consecutively, regardless of which one I add first, the resulting list will still be ordered by date.